roles/openshift_autoheal/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+openshift_autoheal_awx_ca: ""
```

Comment:
add the new insecure?

Reply:
I think it's probably best not to, as we don't want people running insecure in production. People who want to run insecure in development could manually edit the configuration, or use plain http so there's no illusion of security